core/src/main/java/org/elasticsearch/action/bulk/TransportShardBulkAction.java

```diff
@@ -135,7 +135,7 @@ private static BulkItemResultHolder executeIndexRequest(final IndexRequest index
         if (indexResult.hasFailure()) {
             return new BulkItemResultHolder(null, indexResult, bulkItemRequest);
         } else {
-            IndexResponse response = new IndexResponse(primary.shardId(), indexRequest.type(), indexRequest.id(),
+            IndexResponse response = new IndexResponse(primary.shardId(), indexRequest.type(), indexRequest.id(), indexRequest.source().utf8ToString(),
```

Comment:
After looking at the output, is there a way to make it a bit more readable? There are extra `\`s in the output. Maybe you could use `replace("\"," ")`. But there probably is a better way